app/lib/forms/choose_school.rb

```diff
@@ -54,7 +54,7 @@ def possible_institutions
   private
 
     def course
-      @course ||= Course.find(wizard.store["course_id"])
+      @course ||= Course.find_by(name: ::Course::LEGACY_NAME_MAPPING[wizard.store["choose_your_npq"]])
```

Comment:
We're repeating this in a lot of different places, is there any way to move this name to ID translation inside the question and leave all the other lookups the same. Holding this secondary primary key makes things more complex than it has to be.

Reply:
We are actually still setting course_id in that question, so I'm not sure if we need to change these other lookups? We still have course_id.

Reply:
Sorry only just saw this, easier option would be to have in the DB as new name or UI name

Reply:
I think its also because havent really know how important this new text really is